staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go

```diff
@@ -883,7 +880,9 @@ func NewDefaultAPIGroupInfo(group string, scheme *runtime.Scheme, parameterCodec
 // getOpenAPIModels is a private method for getting the OpenAPI models
 func (s *GenericAPIServer) getOpenAPIModels(apiPrefix string, apiGroupInfos ...*APIGroupInfo) (*spec.Swagger, error) {
 	if s.openAPIConfig == nil {
-		return nil, nil
+		// SSA is GA (enabled by default) and requires OpenAPI config to be set
```

Comment:
I agree.  In my opinion, to be a kube-compatible API at this point, it must support SSA. Aggregated apiservers serving their APIs must participate to continue updating past this point.